docs/operators/rln-quickstart.md

```diff
@@ -0,0 +1,70 @@
+# Quickstart: Running a nwaku node with Rate Limiting Enabled (waku-rln-relay)
+
+This guide explains how to run a nwaku node with RLN (Rate Limiting Nullifier) enabled.
+
+[RLN](https://rfc.vac.dev/spec/32/) is a protocol integrated into waku v2, 
+which prevents spam-based attacks on the network.
+
+For further background on the research for RLN tailored to waku, refer
+to [this](https://rfc.vac.dev/spec/17/) RFC.
+
+Registering to the membership group has been left out for brevity.
+If you would like to register to the membership group and send messages with RLN,
+refer to the [on-chain chat2 tutorial](../tutorial/onchain-rln-relay-chat2.md)
```

Comment:
Let's clarify that this is for participation in a specific testnet (it serves as an example of RLN, and is not essential to an RLN guide in general).

Reply:
Addressed in [31b55aa](https://github.com/status-im/nwaku/pull/1311/commits/31b55aab1df804b2d6b63eb989d3e47939de14d1)